src/new-client/src/App/Trades/TradesGrid/SetFilter/SetFilter.tsx

```diff
@@ -25,31 +29,63 @@ export const SetFilter = forwardRef<unknown, IFilterParams>(
 
     const [filterSearchText, setFilterSearchText] = useState("")
     const [selectedFilters, setSelectedFilters] = useState(
-      () => new Set<string>(),
+      () => new Set<string>(filterOptions),
     )
     const inputRef = useRef<HTMLInputElement>(null)
 
+    const resetFilters = useCallback(
```

Comment:
Linter actually complains about removing because it's a dependency of an effect.  Ultimately, we can solve with @josepot recommendation to lift some of the behavior in the useEffects into streams.